src/control/system/raft/database_checker.go

```diff
@@ -74,13 +80,26 @@ func (cdb *CheckerDatabase) updateFinding(finding *checker.Finding) error {
 	return nil
 }
 
+func (cdb *CheckerDatabase) removeFinding(finding *checker.Finding) error {
+	_, found := cdb.Findings[finding.Seq]
```

Comment:
this could be combined with the next line

Reply:
Fixed